i18n-msg.html

```diff
@@ -135,53 +100,7 @@
     },
 
     /**
-     * An object containing the set of known language locales that have been loaded.
-     */
-    locales: {
-      type: Object,
-      value: I18nMsg.locales // static
-    },
-
-    ready: function() {
-      this.messagesUrl = I18nMsg.url;
-      this._setLanguage(I18nMsg.lang);
-
-      // Have instances observe window.I18nMsg.lang changes
-      // and go fetch the localized messages.json file.
-      var observerLang = new PathObserver(window.I18nMsg, 'lang');
-      observerLang.open(function(newValue, oldValue) {
-        _numInstancesUpdated = 0;
-        this._setLanguage(newValue);
-      }.bind(this));
-
-      // Have instances observe window.I18nMsg.url changes
-      // and go fetch the localized messages.json file.
-      var observerUrl = new PathObserver(window.I18nMsg, 'url');
-      observerUrl.open(function(newValue, oldValue) {
-        _numInstancesUpdated = 0;
-        this.messagesUrl = newValue;
-
-        // TODO: this will xhr the json file for each instance.
-        // if (!this.language || _fetching) {
-        //   return;
-        // }
-
-        this._fetchLanguage();
-      }.bind(this));
-
-      _instances.push(this);
-    },
-
-    attached: function() {
-      var msg = this.getMsg(this.msgid)
-      if (msg) {
-        this.msg = msg;
-        this.innerHTML = msg;
-      }
-    },
-
-    /**
-     * Returns a message in the current locale (set by `window.I18nMsg.lang`).
+     * Call parentGetMsg
```

Comment:
This describe feels incomplete. Shouldn't the description be the same as before?